spec/controllers/api/v1/timelines/tag_controller_spec.rb

```diff
@@ -1,41 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails_helper'
-
-describe Api::V1::Timelines::TagController do
-  render_views
-
-  let(:user) { Fabricate(:user) }
-
-  before do
-    allow(controller).to receive(:doorkeeper_token) { token }
-  end
-
-  context 'with a user context' do
-    let(:token) { Fabricate(:accessible_access_token, resource_owner_id: user.id) }
-
-    describe 'GET #show' do
-      before do
-        PostStatusService.new.call(user.account, text: 'It is a #test')
-      end
-
-      it 'returns http success' do
-        get :show, params: { id: 'test' }
-        expect(response).to have_http_status(200)
-        expect(response.headers['Link'].links.size).to eq(2)
-      end
-    end
-  end
-
-  context 'without a user context' do
-    let(:token) { Fabricate(:accessible_access_token, resource_owner_id: nil) }
-
-    describe 'GET #show' do
-      it 'returns http success' do
-        get :show, params: { id: 'test' }
-        expect(response).to have_http_status(200)
-        expect(response.headers['Link']).to be_nil
-      end
-    end
-  end
```

Comment:
According to the [docs](https://docs.joinmastodon.org/methods/timelines/#tag), it shouldn't be possible to make an unauthenticated request to this endpoint if the instance does not allow for public preview of the timelines. So, this test here doesn't seem to be testing the right thing.
I tried reproducing this test using the following before hook:
```rb
before do
  Form::AdminSettings.new(timeline_preview: false).save
end
```
I think that should disable public preview. However, it still allows for unauthenticated requests.

Am I missing something?

Reply:
Should be fixed by  #26237.